.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+labels: feature, request, under discussion
+
+---
+
+## Background
+A clear and concise description of where the limitation lies.
+
+## Feature
+A description of the requested feature.
+
+### *Example:* [Optional]
```

Comment:
I am ok with optional, but I think it is not needed. What is weird me though is that example is surrounded by ** and the proposed solution isn't and that it is on a different level. Why?

Reply:
Good point, I wanted to place the `Example` as a subsection of the `Feature` section, but it just looks a bit odd, I guess.